gitleaves/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 __author__ = """Shekhar Tiwatne"""
 __email__ = 'pythonic@gmail.com'
-__version__ = '0.4.7'
+__version__ = '0.4.8'
```

Comment:
Let's call it 0.5.0 as this is not a patch but a new minor release with a new report.

Reply:
This is something I need to sit with you and understand ( possibly need multiple rounds of feedback on.)
Terribly confused over what constitutes what.

Changed to 0.5